### PR TITLE
Bump to GraphQL.NET v8

### PR DESCRIPTION
**Pull request by Shane32**

### CODE REVIEW

File: src/GraphQL.Conventions/GraphQL.Conventions.csproj [hunk 1]
```diff
@@ -27,8 +27,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="GraphQL.DataLoader" Version="7.8.0" />
-    <PackageReference Include="GraphQL.NewtonsoftJson" Version="7.8.0" />
+    <PackageReference Include="GraphQL" Version="8.0.1-preview-1077" />
```
Comment (Shane32) [inline src/GraphQL.Conventions/GraphQL.Conventions.csproj]: Need to release 8.0.1 due to bug in new naming code

Comment (Shane32) [reply src/GraphQL.Conventions/GraphQL.Conventions.csproj]: https://github.com/graphql-dotnet/graphql-dotnet/pull/4033

Comment (Shane32) [reply src/GraphQL.Conventions/GraphQL.Conventions.csproj]:
```suggestion
    <PackageReference Include="GraphQL" Version="8.0.1" />
```

**Review by tlil — APPROVED** [review]
Thanks for this. LGTM

**Comment by tlil** [comment]
Wanna do the release honours, @Shane32 ? :-)

**Comment by Shane32** [comment]
Sure, if you like :-)

**Pull request merged**